### ISSUE TITLE
[Improvement] Multiple requests?

### ISSUE THREAD

**Issue by gixid192** [IonicaBizau/scrape-it#68]
I'm trying to get something like `promise.all`.

Here is my situation, I want to scrape multiple websites at once. Let's say I want Google and Twitter.

```
const res = [];

scrapeIt("https://google.com", {
    
}, (err, page) => {
    res.push(page)
});

scrapeIt("https://twitter.com", {
    
}, (err, page) => {
    res.push(page)
});

//When all of the above done, do the cb handle for the result.
 
console.log(res);
```

Due to Synchronize, the code above won't work. So I wonder if you could give me a walk through for this issue?
Also, let say I want to get the title from Google but tweet's content from Twitter. Meaning each URL will have a different callback. Is there any way to deal with kind of issue?

Thank you.

**Comment by IonicaBizau** [comment]
Please post this as a question on Stackoverflow, and I will check it out. This is just because of the async nature.

**Comment by gixid192** [comment]
As I said earlier, If I do it by VanillaJS, I can use Promise.all. This is specific for your module, not a general "async" issue. I'm not sure if I should ask this issue on StackOverflow.

Please let me know what you think.

**Comment by IonicaBizau** [comment]
`Promise.all` should work here as well.

**Comment by gixid192** [comment]
Thank you, let me try again.

**Comment by IonicaBizau** [comment]
Ask on Stackoverflow. It will help the community.
